WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailToolbar.swift

```diff
@@ -9,9 +9,9 @@ protocol ReaderDetailToolbarDelegate: AnyObject {
 class ReaderDetailToolbar: UIView, NibLoadable {
     @IBOutlet weak var dividerView: UIView!
     @IBOutlet weak var saveForLaterButton: UIButton!
-    @IBOutlet weak var reblogButton: PostMetaButton!
-    @IBOutlet weak var commentButton: PostMetaButton!
-    @IBOutlet weak var likeButton: PostMetaButton!
+    @IBOutlet weak var reblogButton: UIButton!
```

Comment:
This is safe because I updated this view to use new `UIButton.Configuration` API some time ago. It no longer relies on `imageEdgeInsets` or `titleEdgeInsets`.

<img width="320" alt="Screenshot 2025-04-22 at 1 16 01 PM" src="https://github.com/user-attachments/assets/83f27585-716d-42ff-a80c-da3e364b725a" />